package.json

```diff
@@ -33,5 +20,27 @@
     "Fabien O'Carroll <fabien@allou.is>",
     "Kirill Fomichev <fanatid@ya.ru>"
   ],
-  "license": "MIT"
+  "files": [
+    "index.js"
```

Comment:
Not a fan of this change. The tests are part of the documentation for the package, so they should be published to npm.